File: src/gnuwin32/fixed/rwver.pl
while(<>) {
    s/([0-9]+)\.([0-9]+)\.([0-9]+)\s*//;
    my $minor = $2;
    $minor++ if /Patched/;
    $minor = "0".$minor if $minor < 10;
    $ans = "rw$1$minor$3\n";
}
print $ans;

File: src/gnuwin32/fixed/rwver.R
# keep in step with tools/GETDISTNAME

if(R.version$status == "Under development (unstable)") {
    res <- "R-devel"
} else {
    res <- paste("R-", R.version$major, ".", R.version$minor,
                 tolower(R.version$status) , sep="")
}
cat(res)
